src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -695,7 +718,7 @@ public void testListArchives() throws OpenTokException {
         assertNotNull(archives);
         assertEquals(6, archives.size());
         assertEquals(60, archives.getTotalCount());
-        assertThat(archives.get(0), instanceOf(Archive.class));
```

Comment:
Same as with the `this` modification, just wondering if these need to be a part of this PR or if it's better to hold off until we do a code cleanup later.

Reply:
I figured why not tackle it while I am already in the file; to leave it better then when I how it was before. I figured it was harmless since it won't change the functionality or cause a breaking change. Incremental clean up makes less work later imo